README.md

```diff
@@ -1,25 +1,41 @@
-# powerprotect-automation
 # Dell EMC PowerProtect - Common Flows
-This repository contains Python scripts for common flows in Dell EMC PowerProtect
+This repository contains Use Cases for common flows in Dell EMC PowerProtect Data Manager.
+It composed by 3 main parts.
+- [PowerProtect Data Manager Python Scripts](#powerprotect-data-manager-python-scripts)
+- [PowerProtect Data Manager SDK and Use Cases](#powerprotect-data-manager-sdk-and-use-cases)
+- [PowerProtect Data Manager Postman Collection](#powerprotect-data-manager-postman-collection)
+
 ## Authors
 - **Idan Kentor** - [Dell EMC](https://www.dellemc.com)
 - **Qi Wang** - [Dell EMC](https://www.dellemc.com)
+
 ## Supported Platforms
-- PowerProtect Data Manager 19.3/19.4
-## Prerequisites
-- Python 3.x
+- PowerProtect Data Manager 19.3/19.4, continue working on 19.5/19.6
```

Comment:
Keep 19.3 and 19.4, I'll update it to 19.5 and 19.6 early next week and update the Readme.